mis_builder_cash_flow/report/mis_cash_flow.py

```diff
@@ -84,7 +84,7 @@ def init(self):
             WHERE aml.parent_state != 'cancel'
             UNION ALL
             SELECT
-                fl.id as id,
+                fl.id + 1000000 as id,
```

Comment:
Why do you need this? If there's a conflict of ids, you are only delaying 100k records the conflict, not solving it.